### ISSUE TITLE
The icon is broken for scaled popups

### ISSUE THREAD

**Issue by limonte** [sweetalert2/sweetalert2#974]
From https://github.com/sweetalert2/sweetalert2/issues/855#issuecomment-369559111 

If we change the base font-size of `.swal2-popup`, the icon is broken:

```css
.swal2-popup {
  font-size: 1.5rem;
}
```

Live demo: https://jsfiddle.net/ad3quksn/671/

**Comment by limonte** [comment]
@Kuzcoo if you know how to fix this, feel free to submit a PR.

**Comment by acupajoe** [comment]
```css
.swal2-icon {
  font-size: 1rem;
}
```
https://jsfiddle.net/ad3quksn/669/

**Comment by limonte** [comment]
@acupajoe This is the temporary solution which will fix this issue but undo #932

I applied it temporary in the latest release: https://jsfiddle.net/ad3quksn/672/

But we need to find a proper solution

**Comment by limonte** [comment]
Fixed in #984, thanks again @Kuzcoo 

I also fixed toasts in #994
